Build/Customizations/DatabaseTestResourceFactory.cs

```diff
@@ -0,0 +1,54 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Linq;
+using Customizations.SqlServer;
+using Remotion.BuildScript;
+using Remotion.BuildScript.Test;
+
+namespace Customizations;
+
+public class DatabaseTestResourceFactory : ITestResourceFactory
+{
+  private const string c_imageNameParameter = "ImageName";
+  private const string c_hostNameParameter = "HostName";
+
+  public Databases Database { get; }
```

Comment:
naming: the Type is many, the name is singular?

Reading the test, I can see where this is coming from. Seems like we're stuck with this for the time being..?

Reply:
`Databases` acts like an enum, which is why it is named as plural. That does cause a bit of a naming conflict but the singular here is correct as the resource factory only starts a single database.